lib/lightning_web/views/changeset_view.ex

```diff
@@ -1,23 +1,10 @@
 defmodule LightningWeb.ChangesetView do
+  alias LightningWeb.Components.NewInputs
   use LightningWeb, :view
 
-  @doc """
-  Traverses and translates changeset errors.
-
-  See `Ecto.Changeset.traverse_errors/2` and
-  `LightningWeb.ErrorHelpers.translate_error/1` for more details.
-  """
-  def translate_errors(changeset) do
-    Ecto.Changeset.traverse_errors(changeset, fn {msg, opts} ->
-      Enum.reduce(opts, msg, fn {key, value}, acc ->
-        String.replace(acc, "%{#{key}}", to_string(value))
-      end)
-    end)
-  end
-
```

Comment:
Would we perhaps be able to take this opportunity to move this to CoreComponents instead of NewInputs? We should try and move toward getting NewInputs to not need to exist (or be put in modules that aren't prefixed by 'New' :sweat_smile: